cbor-file-magic.mkd

```diff
@@ -285,7 +285,20 @@ If the eight bytes are a problem on the wire (and they often are if CBOR is bein
 If the programs that read the contents of the file already expect to process all of the CBOR data items in the file (not just the first), then the CBOR Tag Sequence format may be easily retrofitted.
 
 The program involved may throw errors or warnings on the CBOR Tag Sequence if they have not yet been updated, but this may not be a problem.
-If it is, then consideration should be given to CBOR Tag Wrapped.
+
+There are situations where multiple objects may be concatenated into a single file.
+If each object is preceeded by a a CBOR Tag Sequence then there may be multiple Tag Sequences in the file.
+If it is, then consideration should be given to CBOR Tag Wrapped instead.
+
+It is even possible that objects are of different types, and this may be a feature or a bug.
+
+As an example from legacy PEM encoded PKIX certificates, many programs accept a series of PKIX certificates in a single file in order to set up a certificate chain.
+The file would contain not just the End-Entity (EE) certificate, but also any subordinate certification authorities (CA) needed to validate the EE.
+This mechanism actually only works for PEM encoded certificates, and not DER encoded certificates.
+One of the reasons for this specification is to make sure that CBOR encoded certificates do not suffer from this problem.
```

Comment:
I'm not quite sure what "this problem" is, and that's maybe whu I'm not sure that it was a reason for this specification...